stockfish/models.py

```diff
@@ -833,7 +836,7 @@ def will_move_be_a_capture(self, move_value: str) -> Capture:
         starting_square_piece = self.get_what_is_on_square(move_value[:2])
         ending_square_piece = self.get_what_is_on_square(move_value[2:4])
         if ending_square_piece != None:
-            if self._parameters["UCI_Chess960"] == "false":
+            if not self._parameters["UCI_Chess960"]:
```

Comment:
@kieferro Good idea, I'll update the code to do that. If a string is detected, I was thinking of raising an error, and saying in the error message that this value has been updated to type bool in the new release of the library. The user could be told to then give a value for an optional param to the function, specifying whether they still meant to send the data type they did. Do you think this would be the best way to handle it?

Reply:
:+1: